packages/apps/sale/src/components/molecules/CartLineActionCard/CartLineActionCard.tsx

```diff
@@ -16,111 +16,139 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React from 'react';
+import React, {useCallback, useMemo} from 'react';
```

Comment:
Create a new card to manage the catalog view, there are too much differences between the two